contrib/buildrpm/buildrpmLibfabric.sh

```diff
@@ -0,0 +1,368 @@
+#!/bin/bash
+
+# Copyright (c) 2016      Cisco Systems, Inc.  All rights reserved.
+#
+# Written by: Landon Curt Noll, Martin Kontsek
+#
+# Permission to use, copy, modify, and distribute this software and
+# its documentation for any purpose and without fee is hereby granted,
+# provided that the above copyright, this permission notice and text
+# this comment, and the disclaimer below appear in all of the following:
+#
+#       supporting documentation
+#       source copies
+#       source works derived from this source
+#       binaries derived from this source or from derived source
+#
+# Landon Curt Noll, Martin Kontsek DISCLAIMS ALL WARRANTIES WITH REGARD
+# TO THIS SOFTWARE, INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+# AND FITNESS. IN NO EVENT SHALL Landon Curt Noll AND Martin Kontsek BE
+# LIABLE FOR ANY SPECIAL, INDIRECT OR CONSEQUENTIAL DAMAGES OR ANY
+# DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+# WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+# ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+# SOFTWARE.
+
+####################################################################
+#                                                                  #
+#                  Libfabric RPM build script                      #
+#                                                                  #
+####################################################################
+
+####################
+# initialized values
+####################
+export specfile="libfabric.spec"
+export rpmbuilddir="$PWD/rpmbuild"
+export prefix="/usr"
+export noop=""
+export install_in_opt=""
+export create_modulefile=""
+export unpack_spec=""
+export verbose=""
+
+##################################
+# If you need custom configure and
+# rpmbuild options, put them here
+##################################
+export configure_options=""
+export rpmbuild_options=""
```

Comment:
In a similar vein, how are provider specific arguments handled? e.g. if you have to change the directory to libnl, how would the script work?